custom/lib_portal/models/hr_employee.py

```diff
@@ -0,0 +1,8 @@
+
+from odoo import models, fields
+
+
+class HREmployee(models.Model):
+    _inherit = 'hr.employee'
```

Comment:
If the model is not going to be extended there is no need to inherit from it

Reply:
Yeah I agree.But I committed this model for the purpose of adding new methods.